### PR TITLE
access scaner

### PR DESCRIPTION
**Pull request by Lostlarry**
adds tool that allows to learn access requirements of certain objects
works on airlocks, windoors, buttons and id locked mechas

it needs a better sprite
if you think about more things to scan please tell me

:cl:  
rscadd: Adds access scanner for finding access requirements of things
/:cl:

### CODE REVIEW

File: yogstation/code/modules/research/designs/autolathe_designs.dm [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/datum/design/airlock_scaner
+	name = "Airlock scanner"
+	id = "airlock_scaner"
+	build_type = AUTOLATHE | PROTOLATHE
+	materials = list(MAT_METAL = 50, MAT_GLASS = 50)
+	build_path = /obj/item/airlock_scaner
+	category = list("initial","Tools","Tool Designs")
```
Comment (AsV9) [inline yogstation/code/modules/research/designs/autolathe_designs.dm]: `initial`

**Comment by oklews** [comment]
A godsend and a half! will this be for any joe some who can find it, or do engineers start with it or what?

**Comment by Lostlarry** [comment]
it's made in auto lathe or protolathe(engi one i think) so it should be easy to get as sci, cargo or engi
it's also available roundstart

**Comment by alexkar598** [comment]
Why not make it in the after attack to detect and show access?

**Comment by Lostlarry** [comment]
i'm kinda new so i don't know how all functions work
also it would mean bashing the airlock before scanning which is kinda weird

**Comment by MegaEmpirical** [comment]
Yessssss


Will this one work on windoors and buttons?

**Comment by MegaEmpirical** [comment]
Buttons?!

**Comment by Lostlarry** [comment]
read description

**Comment by MegaEmpirical** [comment]
Buttons!

**Pull request merged**